crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -506,6 +506,42 @@ impl OlmMachine {
         self.session_manager.get_missing_sessions(users).await
     }
 
+    /// TODO: docs
+    pub async fn get_encryption_settings(
```

Comment:
Could we either keep those just in the bindings or if you disagree, let's expose the `Store` struct and let users use those methods directly on the store.

Of course we need to make sure we don't expose something we don't want.